configs/cms-xcache/xrootd/lcmaps.db

```diff
@@ -0,0 +1,11 @@
+good = "lcmaps_dummy_good.mod"
+
+gridmapfile = "lcmaps_localaccount.mod"
+              "-gridmap /etc/grid-security/grid-mapfile"
+
+defaultmapfile = "lcmaps_voms_localaccount2.mod"
+                 "-gridmap /usr/share/osg/voms-mapfile-default"
+
+xrootd_policy:
+
+gridmapfile -> good | defaultmapfile
```

Comment:
I'd rather handle this using documentation like we do with our other XRootD installations.

Reply:
Also, why aren't we just using the default that ships with xrootd-lcmaps?

Reply:
cause it has the unl gums stuff that makes no sense for these caches.